igor-web/src/main/java/com/netflix/spinnaker/igor/artifacts/ArtifactService.java

```diff
@@ -37,7 +37,8 @@ public interface ArtifactService {
   default List<String> getArtifactVersions(
       @Nonnull String type, @Nonnull String name, String releaseStatus) {
     List<String> releaseStatuses = new ArrayList<>();
-    releaseStatuses.add(releaseStatus);
+    if (releaseStatus == null) releaseStatuses = null;
```

Comment:
This is more than a refactor.  Can we leave the logic changes for a subsequent PR please?  It also seems like a strange line of code to add...does it do anything?

Reply:
**Change reverted** as this method is not called from unit test.
ArtifactServiceSpec method `"service finds artifact versions"()` called `getArtifactVersions(String, String, List<String>)`
To call same method from ArtifactServiceTest and to resolve the ambiguity issue between the overloaded methods, typecasted null with type `List<String>` in `serviceFindsArtifactVersions()`